crunch_scripts/gatk-split-interval-list.py

```diff
@@ -39,11 +39,11 @@ def prepare_gatk_interval_list_collection(interval_list_coll):
             ref_input = interval_list_f.as_manifest()
             break
     # Create and return a portable data hash for the ref_input manifest
-    # try:
```

Comment:
The diff here looks like it's wrong - in both wtsi-hgi's version (https://github.com/wtsi-hgi/arvados-pipelines/blob/master/crunch_scripts/gatk-split-interval-list.py#L42) and yejinyou's version (https://github.com/yejinyou/arvados-pipelines/blob/master/crunch_scripts/gatk-split-interval-list.py#L42), these files are uncommented.